### PR TITLE
Fix hand teleporter exploit

### PR DESCRIPTION
**Pull request by timothymtorres**
## About The Pull Request
Broken, unpowered, or maint settings on teleporter machinery still allowed a hand tele to function with the machine.

Also fun fact - this bugfix was included in #88559

## Why It's Good For The Game
If something is broken, it shouldn't work...

## Changelog
:cl:
fix: Fix broken teleporter allowing hand tele use
/:cl:

### CODE REVIEW

File: code/game/objects/items/teleportation.dm [hunk 1]
```diff
@@ -179,7 +179,17 @@
...
-		if(computer.power_station?.teleporter_hub && computer.power_station.engaged)
+
+		if(!computer.power_station || !computer.power_station.teleporter_hub)
+			continue
+
+		if((computer.power_station.machine_stat & (NOPOWER|BROKEN|MAINT)) || computer.power_station.panel_open)
+			continue
+
+		if((computer.power_station.teleporter_hub.machine_stat & (NOPOWER|BROKEN|MAINT)) || computer.power_station.teleporter_hub.panel_open)
+			continue
```
Comment (MrMelbert) [inline code/game/objects/items/teleportation.dm:190]: Life hack that apparently I'm the only one who knows about 

`computer.power_station.is_operational`
`computer.power_station.teleporter_hub.is_operational`

Comment (timothymtorres) [reply code/game/objects/items/teleportation.dm:190]: This is a godsend!

Comment (timothymtorres) [reply code/game/objects/items/teleportation.dm]: https://github.com/tgstation/tgstation/pull/88565/commits/d117bbe3fd85fa78faa8400f74ddf7ec889ee0de

**Comment by mc-oofert** [comment]
Balance change

**Comment by Ghommie** [comment]
Merge conflicts.

**Comment by timothymtorres** [comment]
Merge fixed.

File: code/game/objects/items/teleportation.dm [hunk 2]
```diff
@@ -183,10 +183,7 @@
 		if(!computer.power_station || !computer.power_station.teleporter_hub)
 			continue
 
-		if((computer.power_station.machine_stat & (NOPOWER|BROKEN|MAINT)) || computer.power_station.panel_open)
-			continue
-
-		if((computer.power_station.teleporter_hub.machine_stat & (NOPOWER|BROKEN|MAINT)) || computer.power_station.teleporter_hub.panel_open)
```
Comment (SmArtKar) [inline code/game/objects/items/teleportation.dm:189]: This is not a fix, ``is_operational`` explicitly checks for these values - you're doing exactly the same BUT removing the panel open code.

Comment (timothymtorres) [reply code/game/objects/items/teleportation.dm:189]: Looks like this got accidentally merged in:

- #88566

Comment (timothymtorres) [reply code/game/objects/items/teleportation.dm:189]: The initial commit in https://github.com/tgstation/tgstation/pull/88565/commits/73b457f1d0feef9a2585b8d2335259ce904bf79b got applied to both branches

**Pull request closed without merging**